test/name_operations.jl

```diff
@@ -1,3 +1,5 @@
+using SparseArrays
+
```

Comment:
Though this package does seem to try to make files able to run stand alone
So maybe better to finish the job:

```suggestion
using NamedDims
using SparseArrays
using Test
```

Reply:
Ah, sorry, I didn't express myself clearly: What I meant was to comment out the other `include`s as you say.

Perhaps it would be best to put the `using SparseArrays` to the `runtests.jl`?

Reply:
> I guess best practice would be to put the using SparseArrays to the runtests.jl

Yes

> And in this case do you suggest above in order to allow running include("test/wrapper_array.jl")?

Probably not but you can.
This package is still doing it in many places so you can do that, but it is a bit of an aberation and shouldn't be.